src/ResourceManager/Billing/AzureRM.Billing.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.14.0'
+ModuleVersion = '0.15.0'
```

Comment:
this is handled by tooling, no need to change the version

Reply:
Reverted